examples/media/src/search.ts

```diff
@@ -0,0 +1,75 @@
+import algoliasearch from 'algoliasearch/lite';
+import instantsearch from 'instantsearch.js';
+import { singleIndex } from 'instantsearch.js/es/lib/stateMappings';
+
+import {
+  articles,
+  createAuthors,
+  createClearFilters,
+  createDates,
+  configuration,
+  searchBox,
+  createSelectedTopics,
+  stats,
+  topics,
+  seeResults,
+} from './widgets';
+
+const searchClient = algoliasearch(
+  '1QDAWL72TQ',
+  '47700f55d95d23f5a57744b9a027ea83'
+);
+
+const search = instantsearch({
+  searchClient,
+  indexName: 'PROD_algolia_blog',
+  routing: {
+    stateMapping: singleIndex('PROD_algolia_blog'),
+  },
+});
+
+const datesDesktop = createDates({
+  container: '[data-widget="dates-desktop"]',
+  header: 'Date',
+});
+const datesMobile = createDates({
+  container: '[data-widget="dates-mobile"]',
+  header: ' Upload Time',
+});
+const selectedTopicsDesktop = createSelectedTopics({
+  container: '[data-widget="selected-topics-desktop"]',
+});
+const selectedTopicsMobile = createSelectedTopics({
+  container: '[data-widget="selected-topics-mobile"]',
+});
+const clearFiltersDesktop = createClearFilters({
+  container: '[data-widget="clear-filters-desktop"]',
+});
+const clearFiltersMobile = createClearFilters({
+  container: '[data-widget="clear-filters-mobile"]',
+});
+const authorsDesktop = createAuthors({
```

Comment:
On desktop, the Authors refinement are before the Date refinement. On mobile, it's the opposite. I would suggest we always put "Date" first.